lib/features/thread/data/repository/thread_repository_impl.dart

```diff
@@ -63,10 +64,8 @@ class ThreadRepositoryImpl extends ThreadRepository {
         accountId,
         limit: limit,
         sort: sort,
-        filter: emailFilter?.filter,
+        filter: EmailFilterCondition(inMailbox: emailFilter?.mailboxId),
         properties: propertiesCreated);
-
-      yield networkEmailResponse;
```

Comment:
why remove it?

Reply:
because I will return value after cached no need return here.

Reply:
For the first time, we need: reduce the loading time for user. Mails will be loaded as soon as possible

Reply:
If so, the user will have to wait a while, when you finish updating the `cache` to return the data. Why not return it to the user and after updating the `cache` you update the latest list for the user.